rules/gcp_audit_rules/gcp_log_bucket_or_sink_deleted.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+from gcp_base_helpers import gcp_alert_context
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    authenticated = deep_get(
+        deep_get(event, "protoPayload", "authorizationInfo", default=[])[0],
```

Comment:
the default will result in out of index error. 
```suggestion
        deep_get(event, "protoPayload", "authorizationInfo", default=[{}])[0],
```